admin_guide/manage_images.adoc

```diff
@@ -0,0 +1,83 @@
+[[admin-guide-manage-images]]
+= Managing Nodes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[manage-images-overview]]
+== Overview
+
+You can manage
+xref:../architecture/core_components/builds_and_image_streams.adoc#image-streams[images]
```

Comment:
I was struggling with the two, actually. The one I picked spokes a bit about images in the light of ImageStreams, whereas the other one speaks plainly about DockerImages, maybe it would be worth pointing to both?